tasks/HandMeThat.tex

```diff
@@ -40,7 +40,7 @@ \subsection*{Procedure}
 	Then
 	\begin{enumerate}[nosep]
 		\item The operator walks near to the object and points at it.
-		\item The asks as many questions as necessary.
+		\item The robot can asks up to 3 questions as necessary.
```

Comment:
Robot may ask more than 3 questions (waste of time). Question 4th and on won't be penalized anymore.

This makes sense since blind questioning is not allowed (fair play)